lib/routemaster/cache_keys.rb

```diff
@@ -0,0 +1,15 @@
+module Routemaster
+  class CacheKeys
+    attr_reader :url
+
+    KEY_TEMPLATE = 'cache:{url}'
+
+    def initialize(url)
+      @url = url
+    end
+
+    def url_key
+      KEY_TEMPLATE.gsub('{url}', @url)
```

Comment:
TBH I'd even be able to make this a static method so you don't need to instantiate an object to get to it.